reference/tiflash/overview.md

```diff
@@ -0,0 +1,70 @@
+---
+title: TiFlash Overview
+summary: Learn the architecture and key features of TiFlash.
+category: reference
+---
+
+# TiFlash Overview
+
+TiFlash is the key component that makes TiDB essentially an Hybrid Transactional/Analytical Processing (HTAP) database. As a column-based storage extension of TiKV, TiFlash provides both good isolation level and strong consistency guarantee.
+
+In TiFlash, the column-based replicas are asynchronously replicated according to the Raft Learner consensus algorithm. When these replicas are read, the Snapshot Isolation level of consistency is achieved by validating Raft index and multi-version concurrency control (MVCC).
+
+## Architecture
+
+![TiFlash Architecture](/media/tiflash/tiflash-architecture.jpg)
+
+The above figure is the architecture of TiDB in its HTAP form, including TiFlash nodes.
+
+TiFlash provides the column-based storage, with a layer of coprocessors efficiently implemented by ClickHouse. Similar to TiKV, TiFlash also has a Multi-Raft system, which supports replicating and distributing data in the unit of Region (see [Data Storage](https://pingcap.com/blog/2017-07-11-tidbinternal1/) for details).
+
+TiFlash conducts real-time replication of data in the TiKV nodes at a low cost that does not block writes in TiKV. Meanwhile, it provides the same read consistency as in TiKV and ensures that the latest data is read. The Region replica in TiFlash is logically identical to those in TiKV, and is split and merged along with the Leader replica in TiKV at the same time.
```

Comment:
I changed not block reads -> not block writes
Also I added logically before "identical to those in TiKV"